### DISCUSSION TITLE
Select option bug [May 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2116] [Discord Threads]
> 🙋‍♂️ **AK**  
## Found bug. I have selection column and when the option is too much long it will go out of the borders in the gallery view. Apologize about the non-English.. 😅 As a developer I guess it is one of the easiest bugs 😉  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/975028597087674438/unknown.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Would you please send a fix for this 🙂  
#  
> 🙋‍♂️ **AK**  
#### I hope I could find time to try and fix it. I will to my best to free some time for it. I guess I only need to change some css for solving it.  
#  
> 💁 **o1lab**  
#### go for it  
#  
> 💁 **o1lab**  
#### thank you  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/975028597955907606)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
